server/controllers/user.controllers.js

```diff
@@ -121,9 +121,21 @@ const UserController = {
   },
 
   getAllRegisteredUsers(req, res, next) {
-    getAllRegisteredUsers()
+    const page = req.query.page || 1;
```

Comment:
Lets remove the default values here and use the one in the services. Having the default values defined in multiple places can cause errors for us in the future.

```suggestion
    const page = req.query.page;
```

Reply:
You didn't resolve this and didn't add a comment on why you think it is better this way. Can we remove this and use the default in the service?